Dockerfile

```diff
@@ -1,9 +1,9 @@
 FROM python:2.7
-RUN apt-get update && apt-get install -y libjpeg62-turbo-dev postgresql-client nodejs npm
+RUN apt-get update && apt-get install -y libjpeg62-turbo-dev postgresql-client nodejs npm vim
 
 WORKDIR /pontoon
 
-RUN npm install -g yuglify
+RUN npm install -g yuglify babel
```

Comment:
I don't think you need `-g`. Travis should be set up so the default install path is on your PATH. (and if it isn't, it is easy to set it up that way).